lib/tasks/relocate_manual.rake

```diff
@@ -0,0 +1,10 @@
+require "manual_relocator"
+require "logger"
+
+desc <<-EndDesc
+Relocate manual from <from-slug> to <to-slug>
+Both <from-slug> and <to-slug> need to be published manuals
+EndDesc
```

Comment:
Note that only the first line of multi-line descriptions for Rake tasks appear in the output from `rake -T`. In this case the first line makes sense on its own, so I think this is OK. Note also that all lines of the description appear in the output from `rake -D`.